### PR TITLE
(maint) Update sample PXE image/corresponding docs, make servers homogeneous

### PR DESCRIPTION
**Pull request by speedofdark**
This includes updating the README to address needed changes in the
supplied example VM image, that image and changes to the vms.yaml file to
have razor-server come up in VirtualBox instead of VMWare.

Changes to the example VM include changing the base memory to 2048Mb and
specifying boot order to have network first such that rebooting and reinstall
work properly.

This PR is required to support https://github.com/puppetlabs/razor-server/pull/429

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
-  - I've personally used 1.4.3, 1.6.3 in the past and 1.6.5 right now.
+  - I've personally used 1.4.3, 1.6.3 in the past and 1.6.5 right now. Version
+2.2.4 also works.
 3. `vagrant plugin install oscar`
-  - Optional: `vagrant plugin install vagrant-multiprovider-snap`
+  - Optional: `vagrant plugin install vagrant-multiprovider-snap`; this is not
+needed for version 6.x.
 4. `vagrant up`
-  - Optional: `vagrant snap take`
+  - Optional: `vagrant snap take`; this is not needed for version 6.x.
```
Comment (caseywilliams) [inline README.md]: It sounds like it was optional before? What makes it not needed here?

**Comment by speedofdark** [comment]
Snapshot capability is now built into the mainline tool no longer requires
an add-on.

File: README.md [hunk 2]
```diff
@@ -35,10 +37,10 @@
 
 If for some reason you don't want to use the VM definition in `example_pxe_boot_vm` then here are some notes of issues you might run into and how to avoid them.
 
-  - Create a VM with atleast 512MB of RAM
+  - Create a VM with at least 2048MB of RAM.
     - I tried 256MB and it froze during the PXE boot process
   - Under system: Add Network to be enabled in the Boot Order.
-    - It can be last but it needs to be enabled.
+    - It must be first in the Boot Order list for reboot and install to work properly.
```
Comment (smcclellan) [inline README.md]: reboot and *reinstall* to be precise.

File: README.md (hunk 1 above)
Comment (smcclellan) [reply README.md]: Agreed; taking a snapshot is a bit of a time-save for future uses. I don't think this has changed with 6.x.

File: README.md [hunk 3]
```diff
@@ -1,44 +1,47 @@
...
-3. `vagrant plugin install oscar`
-  - Optional: `vagrant plugin install vagrant-multiprovider-snap`
-4. `vagrant up`
-  - Optional: `vagrant snap take`
-4. You can ssh into any of the boxes with `vagrant ssh <box_name>`
+3. Install Vagrant
+  - I've personally used 1.4.3, 1.6.3 in the past and 1.6.5 right now. Version
+2.2.4 also works.
+4. `vagrant plugin install oscar`
+  - Optional: `vagrant plugin install vagrant-multiprovider-snap`; this is not
```
Comment (npwalker) [inline README.md]: I would just remove the instructions for adding multiprovider-snap.  `vagrant snapshot` has been built in long enough we could just set a pre-req of using vagrant 1.8+

Comment (speedofdark) [reply README.md]: Updated.

**Comment by speedofdark** [comment]
I think this is fine as a single commit.

**Comment by smcclellan** [comment]
@speedofdark Can you update the commit message here to reflect the intention of this PR? This looks good-to-go once that is done.

**Comment by speedofdark** [comment]
The current commit message is 

"(RAZOR-1060) Add support for adding Ubuntu Bionic 18.04

Support includes updating the README to address needed changes in the
supplied example VM image, that image and changes to the vms.yaml file to
have razor-server come up in VirtualBox instead of VMWare.

Changes to the example VM include changing the base memory to 2048Mb and
specifying boot order to have network first such that rebooting and reinstall
work properly."

I'm not sure what else I need to clarify? Thanks!

**Comment by smcclellan** [comment]
@speedofdark Ubuntu Bionic 18.04 has nothing to do with pe-razor-vagrant-stack, so doesn't make sense to have in the commit log. The changes here are updating documentation, fixing the `"medium-size"` issue (related to vmware_fusion), and updating the .ova file.

**Comment by speedofdark** [comment]
Updated to not mention Ubuntu.

**Pull request merged**